ios/RCTOneSignal/RCTOneSignalEventEmitter.m

```diff
@@ -130,6 +131,46 @@ + (void)sendEventWithName:(NSString *)name withBody:(NSDictionary *)body {
     }];
 }
 
+RCT_EXPORT_METHOD(setPushToStartToken:(NSString *)activityType 
+                  withToken:(NSString *)token) {
+    @autoreleasepool {
+        NSError* err=nil;
+        [OneSignalLiveActivitiesManagerImpl setPushToStartToken:activityType withToken:token error:&err];
+        if (err) {
+            [OneSignalLog onesignalLog:ONE_S_LL_ERROR message:[NSString stringWithFormat:@"activityType must be the name of your ActivityAttributes struct"]];
+        }
+    }
+}
+
+RCT_EXPORT_METHOD(removePushToStartToken:(NSString *)activityType) {
+    @autoreleasepool {
+        NSError* err=nil;
+        [OneSignalLiveActivitiesManagerImpl removePushToStartToken:activityType error:&err];
```

Comment:
I'm using `OneSignalLiveActivitiesManagerImpl` directly because `OneSignal.LiveActivities.removePushToStartToken` with a string activityType is only implemented there.  I couldn't expose it in the [extension](https://github.com/OneSignal/OneSignal-iOS-SDK/blob/main/iOS_SDK/OneSignalSDK/OneSignalLiveActivities/Source/OSLiveActivitiesExtension.swift), as that is swift only.

The `auroreleasepool` I think was a mistake, removed